### PR TITLE
chore(e2e): rhidp-5094 - [Test automation] restore skipped e2e tests nov2024

### PR DESCRIPTION
**Pull request by gustavolira**
## Description

Please explain the changes you made here.

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by 04kash** [comment]
@gustavolira the tech radar plugin should be rendering Storage and not Process as we are providing data for the tech radar plugin from here:

File: e2e-tests/playwright/e2e/plugins/ocm.spec.ts [hunk 1]
```diff
@@ -32,6 +32,18 @@ test.describe.serial("Test OCM plugin", () => {
...
+    // Wait for the expected path in the URL
+    await page.waitForURL(`**${expectedPath}`, {
+      waitUntil: "domcontentloaded", // Wait until the DOM is loaded
+      timeout: 10000, // Set timeout to 10 seconds
+    });
+
+    expect(page.url()).toContain(expectedPath);
+
+    await uiHelper.verifyHeading("Your Managed Clusters");
+    await uiHelper.verifyText(clusterDetails.clusterName);
```
Comment (subhashkhileri) [inline e2e-tests/playwright/e2e/plugins/ocm.spec.ts:46]: This already being verified in bellow line `await uiHelper.verifyRowInTableByUniqueText(clusterDetails.clusterName`

**Pull request merged**